### ISSUE TITLE
min and max date ranges for calendar

### ISSUE THREAD

**Issue by Hamza-falcon** [acro5piano/react-native-big-calendar#440]
Its not possible to set only a specific date and disable the rest like in react-big-calander

**Comment by acro5piano** [comment]
Can you describe it more in detail?

**Comment by Hamza-falcon** [comment]
i'm trying to show only a range of the month as available and every other date should be disabled. In react-big-calander there are props Min and Max that allow us to only show available date and the rest is greyed out and is not clickable
![Screenshot 2021-06-04 at 10 23 24 AM](https://user-images.githubusercontent.com/76831548/120749984-f977c980-c51e-11eb-8b16-29ca9f78aa55.png)

**Comment by acro5piano** [comment]
That's a good feature. Thanks for sharing. If you send a PR for this, I'm happy to merge!

**Comment by Hamza-falcon** [comment]
i haven't added the feature i'm saying it in react-big-calendar and not in react-native-big-calendar

**Comment by acro5piano** [comment]
> i haven't added the feature i'm saying it in react-big-calendar and not in react-native-big-calendar

Yes, I know it. Currently I can't develop new features for lib much because I don't use it for my current project and I'm busy for my full-time job now.
So if you want the feature and you create a PR for the feature, I'm happy to merge it.

Please remember this is an OSS and I'm working for my full-time job.
